CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ else()
 	add_subdirectory("../../../framework/" "${CMAKE_BINARY_DIR}/framework/")
 endif()
 
-add_executable(Group8FinalProject "src/main.cpp" "src/camera.cpp")
```

Comment:
Had some issues with linker. Some cpp dev might hate me for this, but I just ignored the normal CMAKE way.